### DISCUSSION TITLE
[Proposed nerf revision on SSC, TK]

### DISCUSSION THREAD

**Discussion by Capoguerra** [chromiecraft/chromiecraft#7441] [General]
Only lower the hp of the final bosses (Vashj and Kaelthas) by 25% (nerf = 100*0.75 as per MH) and leave 50% on all trash.

The 25% hp nerf is a great aid to the raid both for players with little equipment and for those more prepared who can face the content in 20players or less.
It is not addictive to face bosses who die too soon, loot loses its value and consequently also the game.
The nerf should simplify the boring/repetitive parts without lowering the difficulty of the raids too much to preserve the mechanics and appeal.

**Comment by heyitsbench** [comment]
> The nerf should [...]

This is incredibly subjective and begs the question of intent. The purpose for the tuning in the first place is pretty cut and dry from [this post on Discord](https://discord.com/channels/788063059926712341/794247581763895316/1203410680355758152) and the [Rates & Changes page](https://www.chromiecraft.com/en/rates-n-changes/):
> Raids are tuned to ensure all mechanics are run and to better reflect the added power players have playing on the 3.3.5a game when going through Vanilla & TBC. Full tuning is only present on current raid releases and will be slowly removed [...]

The reasoning for lifting the tuning however has no intention behind it stated anywhere as far as I'm aware, so to say that the nerf has a particular purpose might not be wrong, but to declare what its goal is is speculative at best. In my opinion, the nerf helps to make the raid more accessible to groups composed of alts, less geared players, or groups with less players in general. In that sense, a blanket lift of tuning makes sense. It also makes these raids take less time for the players who *are* geared with the recently released raid equipment and wastes less of their time if, e.g., they're only after Tsunami Talisman in SSC and still have to make their way through several trash packs to get to Leotheras.

**Comment by elthehablo** [comment]
I was going to write a comment myself, but what Bench said here reflects the full mindset and reasoning behind the lowering of tuning.

At the time, this was also something that was discussed with the community, announced and in our perspective mostly agreed with.

With a lower amount of people running through the lower level raids as they become less relevant, we want them also easily accessible for lower skilled and/or geared players.
